### PR TITLE
Add avatar--spaced modifier

### PR DESCRIPTION
**Pull request by matzimowski**
closes https://github.com/brainly/style-guide/issues/449

🐔 
screen shot 2016-04-14 at 13 46 00

### CODE REVIEW

File: src/components/avatar/_avatar.scss [hunk 1]
```diff
@@ -81,6 +81,10 @@ $includeHtml: false !default;
         top: 10px;
       }
     }
+
+    &--spaced {
+      margin: 0 4px 4px 0;
```
Comment (kdzwinel) [inline src/components/avatar/_avatar.scss]: variable

Comment (matzimowski) [inline src/components/avatar/_avatar.scss]: you are so variable

Comment (matzimowski) [inline src/components/avatar/_avatar.scss]: @kdzwinel up

**Comment by aju** [comment]
let it  be!
:ok:

**Pull request merged**